### DISCUSSION TITLE
Share custom context across multiple QueryClients in a window scope(MFE architecture)

### DISCUSSION THREAD

**Discussion by prasaddola** [TanStack/query#4220] [Q&A]
Is there any working example on how custom context is shared across multiple QueryClients in a window scope(MFE architecture)

**Comment by TkDodo** [comment]
I think @ecyrbe is doing something similar with contextSharing ?

**Reply by ecyrbe** [reply]
No custom context, no. Just using shared context.  
  
But if you want to use custom context and not use shared context, i suggest creating a superset library that does the context instanciation. Then use module federation singleton shared libs to share your lib and context across your micro front-ends.  

No example from me, since i'm not using this.
